pages/common/plocate.md

```diff
@@ -10,4 +10,4 @@
 
 - Look for a file by its exact filename (a pattern containing no globbing characters is interpreted as `*pattern*`):
 
-`plocate */{{filename}}`
+`plocate */{{path/to/file}}`
```

Comment:
```suggestion
`plocate */{{filename}}`
```

Reply:
[Invalid](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) format suggested.

Reply:
Doesn't this example require an exact file name rather than path?

Reply:
I guess that `*/` can be removed and `{{path/to/file}}` can be used. I should check it.

Reply:
Changes will be applied: my comment was wrong. Unresolving....